libstuff/SLog.cpp

```diff
@@ -68,6 +69,7 @@ static const set<string> PARAMS_WHITELIST = {
     "policyID",
     "reportID",
     "requestTimestamp",
+    "secondaryLogin"
```

Comment:
shouldn't we have a comma at the end here?
```suggestion
    "secondaryLogin",
```